### ISSUE TITLE
We have multiple BIC definitions for the same bank code which makes Schwifty selecting the wrong one for the IBAN.

### ISSUE THREAD

**Issue by Natim** [mdomke/schwifty#153]
```
>>> BIC.candidates_from_bank_code('FR', '30004')

[<BIC=BNPAFRPPIFN>, <BIC=BNPAFRPPPAA>, <BIC=BNPAFRPPMED>, <BIC=BNPAFRPPCRN>, <BIC=BNPAFRPP>, <BIC=BNPAFRPPPAE>, <BIC=BNPAFRPPPBQ>, <BIC=BNPAFRPPNFE>, <BIC=BNPAFRPPPGN>, <BIC=BNPAFRPPXXX>, <BIC=BNPAFRPPBOR>, <BIC=BNPAFRPPCRM>, <BIC=BNPAFRPPPVD>, <BIC=BNPAFRPPPTX>, <BIC=BNPAFRPPPAC>, <BIC=BNPAFRPPPLZ>, <BIC=BNPAFRPP039>, <BIC=BNPAFRPPENG>, <BIC=BNPAFRPPNEU>, <BIC=BNPAFRPPORE>, <BIC=BNPAFRPPPEE>, <BIC=BNPAFRPPPXV>, <BIC=BNPAFRPPIFO>]
```

**Comment by Natim** [comment]
![image](https://github.com/mdomke/schwifty/assets/229453/42c4d1e0-e7b1-4d6a-ad00-8ffd3f94397b)

**Comment by mdomke** [comment]
Fixed in release 2023.09.0
